ansible/playbooks/roles/compatibility_layer/tasks/add_overlay.yml

```diff
@@ -58,6 +58,28 @@
         selectattr('eclass-overrides', 'equalto', True) | map(attribute='name') | join(' ')
       }}
 
+- name: Use git instead of rsync for the Gentoo overlay
+  copy:
+    dest: "{{ gentoo_prefix_path }}/etc/portage/repos.conf/gentoo.conf"
+    mode: "0644"
+    content: |
+      [DEFAULT]
+      main-repo = gentoo
+      sync-git-pull-extra-opts = --quiet
+      
+      [gentoo]
+      priority  = 1
+      location  = {{ gentoo_prefix_path }}/var/db/repos/gentoo
+      sync-uri  = https://github.com/gentoo/gentoo.git
```

Comment:
If you want something with pre-generated metadata, etc, you can use https://github.com/gentoo-mirror/gentoo.git instead. The repo above is where development happens (well, a mirror of it), so it won't have the metadata, news items, glsa-check, etc. You can recover those using the scripts from https://github.com/hasufell/portage-gentoo-git-config, though, if you want.

Reply:
Ah okay. I just added some steps to add those scripts, but then it sounds like it's easier to use the mirror repo. Is there any downside in using that one?

Reply:
No, it's fine, that does the steps you added. However, if you develop gentoo itself, then it makes more sense to use `gentoo/gentoo` instead of the mirror, because you can commit and push directly rather than having to develop on a separate place. If you already added the steps, though, you may as well leave this as is, because you can use the steps to easily generate metadata for the eessi overlay as well.

Reply:
Okay, I hadn't enabled the scripts for additional overlays yet, but then I'll do that. Thanks!